### DISCUSSION TITLE
求助，无法从模板插入

### DISCUSSION THREAD

**Discussion by xiaomusq** [windingwind/zotero-better-notes#485] [Q&A] [answered]
![image](https://github.com/windingwind/zotero-better-notes/assets/26929299/57490a3f-df78-47bd-bd1d-791d7f5b7e06)

**Comment by xinchen0430** [comment]
同问

**Comment by windingwind** [comment, marked as answer]
link不再是预定义变量。请用`Zotero.BetterNotes.api.convert.note2link(topItem)`获取
